networkx/algorithms/tests/test_matching.py

```diff
@@ -603,3 +603,570 @@ def test_wrong_graph_type(self):
         raises(error, nx.maximal_matching, nx.MultiGraph())
         raises(error, nx.maximal_matching, nx.MultiDiGraph())
         raises(error, nx.maximal_matching, nx.DiGraph())
+
+
+class TestCountPlanarPerfectMatchings:
+    """Unit tests for the
+    :func:`~networkx.algorithms.matching.kasteleyn_orientation`.
+
+    """
+
```

Comment:
I think this will fix the ci failures
```suggestion
    pytest.importorskip("numpy")
```

Reply:
Thank you for the help! I got all the tests to pass.